library/src/test/java/bio/terra/pfb/LibraryTest.java

```diff
@@ -1,14 +1,85 @@
 package bio.terra.pfb;
 
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 
+@ExtendWith(MockitoExtension.class)
 class LibraryTest {
+  private static final String TEST_FILE_LOCATION = "path/to/file/minimal_schema.avro";
+  private static final String FAKE_DATA_RETURN = "Fake data return";
+  @Mock PfbReader pfbReader;
 
   @Test
-  void testStubMethod() {
-    assertThat(Library.getNumber5(), equalTo(5));
+  void showSchemaTest() throws IOException {
+    when(pfbReader.showSchema(TEST_FILE_LOCATION)).thenReturn(FAKE_DATA_RETURN);
+    Library library = new Library(pfbReader);
+    assertReturnValueMatches(library.showSchema(TEST_FILE_LOCATION));
```

Comment:
These are just verifying that the library shows the same info as the pfbreader?

Reply:
Yeah - I suppose I'm confirming that the library methods are wired up to the correct pfbreader methods and that the data/file location is correctly getting passed through. 
The sonarcloud code coverage metric might be gamifying this to much... haha Mostly kidding -- Good to have sanity checks in place that things are wired up correctly.